typings/objection/index.d.ts

```diff
@@ -887,6 +887,7 @@ declare namespace Objection {
     resultSize(): Promise<number>;
 
     page(page: number, pageSize: number): QueryBuilder<QM, Page<QM>>;
+    range(): QueryBuilder<QM, Page<QM>>;
```

Comment:
shouldn't start and end parameters just be marked optional for that?

Reply:
If they were marked as optional then the typings would allow `.range(10)`, but that's not something objection will handle internally.

Reply:
Makes sense.